@PTOptions/PTOptions.m

```diff
@@ -212,15 +212,16 @@
          end
       end   
       
-      function set.swapsPerIter(this, value)
+
+      function this = set.swapsPerIter(this, value)
          if(value == floor(value) && value > 0)
             this.swapsPerIter = lower(value);
          else
             error(['Please enter a positive integer for the swaps per iteration.']);
          end
       end 
       
-      function set.temperatureEta(this, value)
+      function this = set.temperatureEta(this, value)
          if(value == floor(value) && value > 0)
             this.temperatureEta = lower(value);
          else
```

Comment:
Again, changes in this file are due to switching the inheritance from
matlab.mixin.SetGet to matlab.mixin.CustomDisplay